viewing_party/party.py

```diff
@@ -1,23 +1,119 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    inputs = [title, genre, rating]
+    for input in inputs:
+        if input == None:
+            return None
+    new_movie = {}
+    new_movie["title"] = title
+    new_movie["genre"] = genre
+    new_movie["rating"] = rating
+    return new_movie
+
+def add_to_watched(user_data, movie): 
+    user_data["watched"].append(movie)
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+
+def watch_movie(user_data, movie_title):
+    movie_index = 0
+    for movie in user_data["watchlist"]:
+        if movie_title == movie["title"]:
+            user_data["watched"].append(movie)
+            user_data["watchlist"].pop(movie_index)

```

Comment:
Nice! Adding the counter lets you rightly use `pop` with the counter as an index, however you can also use `remove` with the `movie`. 
```python
user_data["watchlist"].remove(movie)
```
This is safe to use since we know for sure that `movie` is in the list we're removing it from.